docs/configure

```diff
@@ -478,10 +480,11 @@ ditch_maxmsgsize:
 		and domain = '${quote_mysql:$domain}' \
 		and users.maxmsgsize > 0 \
 		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
-  local_part_suffix = -*
-  local_part_suffix_optional
+#  local_part_suffix = -*
+#  local_part_suffix_optional
```

Comment:
On Sep 17, 2015 12:37 PM, Udera <notifications@github.com> wrote:
>
> In docs/configure:
>
> > @@ -478,10 +480,11 @@ ditch_maxmsgsize:
>
> >  		and domain = '${quote_mysql:$domain}' \
>
> >  		and users.maxmsgsize > 0 \
>
> >  		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
>
> > -  local_part_suffix = -*
>
> > -  local_part_suffix_optional
>
> > +#  local_part_suffix = -*
>
> > +#  local_part_suffix_optional
>
> Ok, let's keep the local_part_suffix in the configuration, I will go through the conf.d/-files and add these.
>
> Is + the more common suffix? Or we use both?
>
> local_part_suffix = +* : -*
I have often seen hyphens within an address local part, not as a suffix. In my experience the only common sufffix delimiter is the plus sign.
By the way, I'm new to this list but have used Vexim for ages. I signed on as I'm preparing to rebuild/update my servers. I was afraid Vexim would have gone dormant, but I'm really happy to see such a vibrant community! I did some local patches to add features. I'll review these against the current version and contribute any that are still useful.
Scott